org.eclipse.jdt.ls.core/META-INF/MANIFEST.MF

```diff
@@ -34,8 +34,9 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.xtext.xbase.lib,
  org.eclipse.core.filesystem;bundle-version="1.7.0"
 Export-Package: org.eclipse.jdt.ls.core.internal;x-friends:="org.eclipse.jdt.ls.tests",
- org.eclipse.jdt.ls.core.internal.contentassist;x-friends:="org.eclipse.jdt.ls.tests",
  org.eclipse.jdt.ls.core.internal.commands;x-friends:="org.eclipse.jdt.ls.tests",
+ org.eclipse.jdt.ls.core.internal.contentassist;x-friends:="org.eclipse.jdt.ls.tests",
+ org.eclipse.jdt.ls.core.internal.corext.codemanipulation,
```

Comment:
is this import really necessary?

Reply:
We use org.eclipse.jdt.ls.core.internal.corext.codemanipulation.OrganizeImportsOperation.

Reply:
ok but please add x-friends